src/assets/js/src/utils.js

```diff
@@ -116,7 +116,7 @@ notify = {
   }
 }
 
-validateNotAllEmpty = message => {
+function validateNotAllEmpty(message) {
```

Comment:
Why do that manually, can't babel do it?

Reply:
This is not for backward compatibility but for coherence of the code.
global functions were written and functions so far in the code (and here it appears as an undeclared global variable)